config/locales/en.yml

```diff
@@ -52,6 +53,10 @@ en:
       User registration is disabled on this system. Please ask an administrator to create an
       account for you.
     login_with_auth_provider: "or sign in with your existing account"
+    signup_with_auth_provider: "or sign up using"
```

Comment:
You might want to change that to include the auth provider name variable, because the translation may be more explicit then.